test/Sema/dynamic_self_implicit_conversions.swift

```diff
@@ -1,5 +1,6 @@
 // RUN: %target-swift-frontend -typecheck -dump-ast %s | %FileCheck %s
 
+// FIXME: Make this a SILGen test instead.
```

Comment:
Is there more to this than just moving the test to test/SILGen and redoing the CHECK: lines to look for SIL instructions?